.github/component-owners.yml

```diff
@@ -0,0 +1,32 @@
+---
+# Each component identified by its path prefix has a list of owners
+components:
+
+  # Ownership applies recursively to any file in a directory
+
+  ./:
```

Comment:
@StaticRocket probably we can drop this since CODEOWNERS file would handle it
(assuming there's no clash between `CODEOWNERS` & `dyladan/component-owners@main`)